src/index.html

```diff
@@ -1,15 +1,189 @@
+          <input
+            type="number"
+            name="old"
+            id="old"
+            value="12"
+            min="0"
+            max="300"
+          >
+        </label>
+        <br>
+        <label class="form-field" for="Full-date-of-birth">
+          Full date of birth:
+          <input
+            type="date"
+            name="birthdate"
+            id="Full-date-of-birth"
+          >
+        </label>
+        <br>
+        <label class="form-field" for="term-of-the-agreement">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="terms"
+            id="term-of-the-agreement"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="set">
+        <legend>Registration</legend>
+        <label class="form-field" for="email">
+          E-mail:
+          <input
+            name="email"
+            placeholder="email@example.com"
+            type="email"
+            id="email"
+          >
+        </label>
+        <br>
+        <label class="form-field" for="password">
+          Password:
+          <input
+            name="password"
+            type="password"
+            id="password"
+            minlength="8"
+            maxlength="16"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="set">
+        <legend>An interesting fact about you!</legend>
+        <div class="form-field">
+          Do you love cats?
+          <label for="Do-you-love-cats">
+            <input
+              name="cats"
+              type="radio"
+              id="Do-you-love-cats"
+              value="y">
+                Yes
+          </label>
+          <label for="cats">
+            <input
+              name="cats"
+              type="radio"
+              id="cats"
+              value="n"
+            >
+              No
+          </label>
+        </div>
+        <br>
+        <label class="form-field" for="What-is-your-f-color">
+          What is your favorite color?
+          <input
+            type="color"
+            name="color"
+            id="What-is-your-f-color"
+            value="f000"
+          >
+        </label>
+        <br>
+        <label class="form-field" for="time">
+          What time do you go to bed?
+          <input
+            type="time"
+            id="time"
+            name="time"
+          >
+        </label>
+        <br>
+        <label class="form-field" for="cars">
+          What are your favorite brand of cars?
+          <select
+            name="car"
+            id="cars"
+            multiple
+          >
+            <option value="">BMW</option>
+            <option value="">Audi</option>
+            <option value="">Lada</option>
+          </select>
+        </label>
+        <br>
+        <label class="form-field" for="slider">
+          How do you rate our work?
+          <input
+            type="range"
+            name="slider"
```

Comment:
```suggestion
            name="work-rate"
```
Try to give meaningful names